update-vo-config/update-vo-config

```diff
@@ -0,0 +1,507 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 CNRS and University of Strasbourg
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import argparse
+from operator import itemgetter
+import os
+import re
+import sys
+import xml.etree.ElementTree as ET
+from OpenSSL import crypto
+import urllib2
+
+SW_MANAGER_ROLES = ['/Role=lcgadmin', '/admin', '/Role=swadmin',
+                    '/Role=sgmadmin', '/Role=sgm', '/Role=SoftwareManager',
+                    '/Role=VO-Software-Manager', '/Role=SW-Admin']
+
+PRODUCTION_ROLES = ['/Role=production', '/Role=Production', '/Role=prod',
+                    '/Role=ProductionManager']
+
+PILOTE_ROLES = ['/Role=pilot']
+
+FQAN_TYPE_MAPPING = {
+    'pilot': 'pilot',
+    'production': 'p',
+    'SW manager': 's'
+}
+
+VO_CARD_URI = "http://operations-portal.egi.eu/xml/voIDCard/public/all/true"
+CERT_DIR = 'vo/certs'
+PARAMS_DIR = 'vo/params'
+SITE_PARAMS_DIR = 'vo/site'
+# The base XML element of the VO Card
+ROOT_ELEM = "VoDump"
+ID_CARD_ELEM = "IDCard"
+ID_ATTR = "CIC_ID"
+
+DESCRIPTION = "Configure VO-related PAN templates"
+VERSION = "0.1"
+
+
+class VOMSEndpoint(object):
+    """A class representing a VOMS server endpoint (used by a specific VO)
+    """
+    def __init__(self):
+        self.hostname = None
+        self.https_port = 8443
+        self.vomses_port = 0
+        self.old_cert_retrieved = False
+        self.cert = None
+        self.old_cert = None
+        self.is_admin = True
+
+    def get_cert(self):
```

Comment:
If this does nothing, what is it for?

Reply:
The useless function has been removed, as well as a useless class attribute.